snap/snapcraft.yaml

```diff
@@ -60,24 +58,12 @@ parts:
     source-type: git
     source-tag: "v0.40.4"
     build-snaps:
-      - go
+      - go/1.22/stable
     build-packages:
       - build-essential
       - libsystemd-dev
       - libbpfcc-dev
       - bpfcc-tools
-      - on amd64 to arm64:
-        - gcc-aarch64-linux-gnu
-        - linux-libc-dev-arm64-cross
-        - libc6-dev-arm64-cross
-    build-environment:
-      - to amd64:
-        - GOOS: linux
-        - GOARCH: amd64
-      - to arm64:
-        - GOOS: linux
-        - GOARCH: arm64
-        - CC: /usr/bin/aarch64-linux-gnu-gcc
```

Comment:
This diff is a revert of a commit that was accidentally pushed directly to main.